src/todolist.js

```diff
@@ -0,0 +1,40 @@
+let todoList = localStorage.getItem('todoList') ? JSON.parse(localStorage.getItem('todoList')) : [];
+const getTodoList = () => todoList;
+const saveTodoList = () => {
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+};
+const addItem = (task) => {
+  if (task) {
+    const taskItem = {
+      description: task,
+      completed: false,
+      index: todoList.length + 1,
+    };
+    todoList.push(taskItem);
+    saveTodoList();
+  }
+};
+
+const removeItem = (index) => {
+  todoList.splice(index, 1);
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+};
```

Comment:
- [ ]  As suggested by the previous reviewer, the index values of the remaining tasks should be updated to reflect their positions in the array. If there are 5 tasks, and I remove anyone, regardless of position, the remaining index values should still be 1, 2, 3 , 4 etc respectively. As you see in the images below, instead of 3,4,5,6, the index values should be 1,2,3,4 ☺️.
Before removing the tasks 😎 
![image](https://github.com/Jessekay/ToDo-LIST/assets/99394374/1f9c3512-e877-4309-a827-0ae25a0ec7f0)

After removing the tasks 😎 
![image](https://github.com/Jessekay/ToDo-LIST/assets/99394374/30eefe2b-b635-4c55-a861-cd8c46439068)